LICENSE

```diff
@@ -583,7 +583,7 @@ to choose that version for the Program.
 
   Later license versions may give you additional or different
 permissions.  However, no additional obligations are imposed on any
-author or copyright holder as a result of your choosing to follow a
+author or copyright holder as a preparationResult of your choosing to follow a
```

Comment:
What happened here?

Reply:
Idea being stupid. I renamed a variable. Missed this change